assets/app/views/images.html

```diff
@@ -10,30 +10,53 @@
           <em>{{emptyMessage}}</em>
         </div>
       </div>
-      <div ng-repeat="imageStream in imageStreams">
-        <div class="tile" ng-repeat="tag in imageStream.status.tags">
+      <div ng-repeat="imageStream in imageStreams track by (imageStream | uid)" class="tile">
+        <h2>
+          {{imageStream.metadata.name}}
+        </h2>
+        <div ng-if="imageStream.spec.dockerImageRepository" class="small">Docker image repository: {{imageStream.spec.dockerImageRepository}}</div>
+        <div ng-if="!(imageStream | annotation : 'openshift.io/image.dockerRepositoryCheck') && !imageStream.status.tags.length"
```

Comment:
If the sync hasn't happened yet, should we still show the status tags, but put the "not synced" text after each one?